packages/amesos2/test/solvers/MUMPS_UnitTests.cpp

```diff
@@ -322,6 +322,79 @@ namespace {
     TEST_COMPARE_FLOATING_ARRAYS( xhatnorms, xnorms, 0.005 );
   }
 
+  TEUCHOS_UNIT_TEST_TEMPLATE_3_DECL( MUMPS, SolveTwice, SCALAR, LO, GO )
```

Comment:
It's supposed to be possible to build Amesos2 without Tpetra support, so you might need to protect this unit test with an appropriate macro, depending on the CMakeLists.txt settings.

Reply:
We discussed this more above.